dask/dataframe/core.py

```diff
@@ -6029,7 +6390,15 @@ def is_broadcastable(dfs, s):
     )
 
 
-def elemwise(op, *args, meta=no_default, out=None, transform_divisions=True, **kwargs):
+def elemwise(
+    op,
+    *args,
+    meta=no_default,
+    out=None,
+    transform_divisions=True,
+    partition_metadata=None,
```

Comment:
Who is responsible for making sure that `meta` and `partition_metadata` are in sync? For more internal APIs such as this, I'd probably want to just replace `meta` with `partition_metadata`.